### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by pzrsa**

### CODE REVIEW

**Comment by pzrsa** [comment]
so i've looked at trying to have the default values set for the categories (like household or fridge) and it seems like i cant do that with Prisma. what we could do is set those values when the list is created, we can talk more about this later when the time comes for generating a list.

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+model List {
+  id      Int    @id @default(autoincrement())
+  ownerId Int
+  items   Item[]
+}
+
+model Item {
+  id     Int    @id @default(autoincrement())
+  name   String
+  listId Int
```
Comment (frontendowl) [inline prisma/schema.prisma]: we need category_id column as well

File: prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  list   List   @relation(fields: [listId], references: [id])
+}
+
+model Category {
+  id      Int           @id @default(autoincrement())
+  name    String
+  orderId Int
```
Comment (frontendowl) [inline prisma/schema.prisma]: why do we need orderId column here?

**Review by frontendowl — CHANGES_REQUESTED** [review]
Great job, but, please, add a column for category_id to the Items table.
What would we store in the order_id column of Category table?

**Review by frontendowl — APPROVED** [review]

**Pull request merged**